docs/sql/mysql/onboarding/create-schema.sql

```diff
@@ -174,6 +174,13 @@ CREATE TABLE IF NOT EXISTS audit_log (
     build_time         TIMESTAMP NULL
 ) ENGINE=InnoDB AUTO_INCREMENT=1 CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
 
+CREATE TABLE audit_param (
+    audit_log_id       VARCHAR(36),
+    timestamp_created  TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    param_key          VARCHAR(256),
+    param_value        VARCHAR(3072)
```

Comment:
3072 differs from 4000 for other databases but I guess that mysql is not preferred anyway.

Reply:
Yes, this value is intentional. Indexed varchar on MySQL has max. length of 3072.